libraries/loggly_helpers.rb

```diff
@@ -0,0 +1,41 @@
+module LogglyHelpers
+  def configure_files(original_files)
+    valid_files(original_files).map(&:to_h).map do |file|
```

Comment:
I think it's probably an artifact from when I refactored this recipe from `apetresc`.